extensions/sql-database-projects/src/test/baselines/SSDTProjectAfterUpdateBaseline.xml

```diff
@@ -69,4 +69,8 @@
     <Folder Include="Views\User"/>
     <Build Include="Views\User\Profile.sql"/>
   </ItemGroup>
-<Import Condition="'$(NetCoreBuild)' == 'true'" Project="$(NETCoreTargetsPath)\Microsoft.Data.Tools.Schema.SqlTasks.targets"/><ItemGroup><PackageReference Condition="'$(NetCoreBuild)' == 'true'" Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All"/></ItemGroup></Project>
+  <Import Condition="'$(NetCoreBuild)' == 'true'" Project="$(NETCoreTargetsPath)\Microsoft.Data.Tools.Schema.SqlTasks.targets"/>
```

Comment:
great! the add file tests baselines don't need update?

Reply:
Not with the way the current tests function.  They currently load a sqlproj, add files, save, reload the sqlproj, and verify via the reloaded project instead of doing a string comparison on the edited sqlproj.